src/battle_records.c

```diff
@@ -311,10 +320,14 @@ static void PrintLinkBattleRecord(struct LinkBattleRecord *record, u8 y, s32 lan
         AddTextPrinterParameterized(gRecordsWindowId, FONT_NORMAL, gStringVar1, 176, (y * 8) + 1, 0, NULL);
     }
 }
+#endif //FREE_LINK_BATTLE_RECORDS
 
 void ShowLinkBattleRecords(void)
 {
-    s32 i, x;
+    s32 x;
```

Comment:
probably should just comment out the whole file if the config is on

Reply:
There's references to these functions and `gRecordsWindowId` in other files too, though. Should I config out the entire file and then add configs in said functions from other files too or leave it as is?